### ISSUE TITLE
Axe tests are showing false positives

### ISSUE THREAD

**Issue by escherina** [noopkat/electric-io#147]
We use `axe-core` in the component unit tests to test for accessibility related issues with the code. However, they seem to always pass, even when the component contains accessibility violations.

Example: in `CardForm.vue`, un-nest the `<input>` from the `<label>` element [on line 10](https://github.com/noopkat/electric-io/blob/master/public/js/components/CardForm.vue#L10) so they are no longer associated, and run `npm test`. The accessibility test should fail, but it is passing.

We should fix the unit tests that use `axe-core` to ensure that they can fail on inaccessible components.

(Credit to @kleinfreund for spotting this - feel free to add anything I missed from your investigation into this issue! 😎)
